app/models/rating.rb

```diff
@@ -0,0 +1,22 @@
+class Rating < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :film
+
+  validates :value, :user_id, :film_id, presence: true
+  validate :rating_value
```

Comment:
If you use custom validators, give them meaningful names like `:rating_value_in_range` so that the reader has a sense from the name of what the test is